Do not apply the setuid permission bit on the vpsm script

See https://github.com/void-linux/void-packages/issues/32156
see https://github.com/sineto/vpsm/pull/2


--

--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@
 
 .PHONY: install
 install:
-	install -Dm4755 vpsm $(DESTDIR)$(PREFIX)/bin/vpsm
+	install -Dm755 vpsm $(DESTDIR)$(PREFIX)/bin/vpsm
 	install -Dm644 man/vpsm.1 $(DESTDIR)/$(PREFIX)/share/man/man1/vpsm.1
 
 .PHONY: uninstall
